### PR TITLE
update crio man page.

### PR DESCRIPTION
**Pull request by Chenditang**
- What I did
update crio man page.

- How I did it

- How to verify it

### CODE REVIEW

File: docs/crio.8.md [hunk 1]
```diff
@@ -74,10 +76,22 @@ crio [GLOBAL OPTIONS] config [OPTIONS]
...
 
+**--default-capabilities**="": capabilities to add to the containers.
+
+**--default-mounts**="": add one or more default mount paths in the form host:container (deprecated)
+
+**--default-runtime**="": default OCI runtime from the runtimes config.
+
+**--default-sysctls**="": sysctls to add to the containers.
+
+**--default-ulimits**="": ulimits to apply to conatainers by default (name=soft:hard)
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md]: conatainers to containers

**Comment by rhatdan** [comment]
@Chenditang Are you still working on this?

**Comment by umohnani8** [comment]
Taking over in #2126 since haven't heard back from @Chenditang

**Pull request closed without merging**